### ISSUE TITLE
Shouldn't "lua" package be renamed to "lua51"?

### ISSUE THREAD

**Issue by xaizek** [Ponce/slackbuilds#213]
Otherwise both Slackware current and SBo current have "lua" but rather different ones (5.1 vs 5.4, which are incompatible).  Similarly, maybe Slackware shouldn't call the package "lua" and use "lua54" as well.

I came across this while trying to build `lgi` that apparently doesn't support 5.4 (at least its latest release doesn't), updated SlackBuild to do:
```bash
make PREFIX=/usr CFLAGS="$SLKCFLAGS -I/usr/include/lua5.3/" LUA_VERSION=5.3
make install DESTDIR=$PKG LUA_VERSION=5.3
```
Which worked, but then I tried to build `awesome` and hit that it uses `/usr/include/lua.h` as well as `lua` executable.  Managed to build by adding this to `cmake` invocation and `lua -> lua5.3` symlink to `$PATH`:
```bash
-DLUA_LIBRARY=/usr/lib64/liblua5.3.so \
-DLUA_INCLUDE_DIR=/usr/include/lua5.3/ \
```

These are just examples, hence not PRs. There are at least [63 users](https://slackbuilds.org/advsearch.php?q=lua&stype=revdep1) of `lua` package (at least `awesome` isn't there despite it requiring `lua` for build, but it normally gets `lua` due to transitivity).  Just installing "lua" from SBo on top of Slackware's "lua" will probably work (e.g., `wireplumber` should still use the correct `liblua`), but is that what's recommented here?  This from the [wiki](https://github.com/Ponce/slackbuilds/wiki):
> note that I remove from the repository whatever gets added to Slackware current (also under a different name - for example six as python-six)

Makes me think that it's not intentional.

**Comment by Ponce** [comment]
yes, it should, but it's not something that could be done in this repository (as it would lead to a non-maintenable situatio), it has to be done in the main SlackBuilds.org repository.

**Comment by xaizek** [comment]
Maybe there should be a Wiki page outlining such things.  `wireplumber` is also still there despite being now part of `pipewire`.

**Comment by Ponce** [comment]
that of lua is actually a very special case.
usually I remove from this repository what is added to Slackware during current's development cycle but I might miss something, especially if something gets added as part of other packages, so I base this patches also on the reports I receive: so thanks for yours about wireplumber, it will be removed from here at the next global update (usually in the weekend)!

**Comment by Ponce** [comment]
hi @xaizek ,

B. Watson, one of the other admins of SlackBuilds.org worked a lot on this and it's now merged: the lua package tracks the 5.4.x version (like in current) while the lua51 package is dedicated to the 5.1.x version.
you shouldn't have any issues building and running awesome now: please let me know!

**Comment by xaizek** [comment]
Hello,

Works fine for me without any modifications now, thanks!
